Tools/maneuver.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+echo $script_dir
+if [[ $1 == "-h" ]]; then
+    echo "Usage: maneuver [maneuver-name]"
+    exit 1
+fi
+
+build_dir=$script_dir/Maneuvers/build
+
+# check if build folder exists
+if [[ ! -d $build_dir ]]; then
+    echo "No build folder present"
+    echo "Do you want build Maneuvers?"
+    read -p "Continue (y/n)?" answer
+
+    if [[ $answer == [Yy] ]]; then
+        mkdir $build_dir
```

Comment:
Maybe first check if the submodule is checked out properly otherwise this will fail and the script continues.